huggingface_keras_imdb/huggingface_keras_imdb.ipynb

```diff
@@ -0,0 +1,297 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5de38601",
+   "metadata": {},
+   "source": [
+    "# Finding Label Errors in IMDB Reviews Dataset using Keras and HuggingFace Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a3c996fb",
+   "metadata": {},
+   "source": [
+    "This example demonstrates the use of a cleanlab-compatible Keras classifier (and pretrained bert models from HuggingFace) to find issues in the IMBD Reviews Dataset and train an improved classifier model using `cleanlab`'s CleanLearning. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e4a75294",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import tensorflow as tf\n",
+    "import tensorflow_datasets as tfds\n",
+    "from transformers import AutoTokenizer, TFAutoModel\n",
+    "from transformers import logging\n",
+    "from sklearn.metrics import accuracy_score\n",
+    "import os\n",
+    "\n",
+    "from cleanlab.experimental.keras import KerasWrapperModel\n",
+    "from cleanlab.classification import CleanLearning\n",
+    "\n",
+    "logging.set_verbosity(40)\n",
+    "os.environ['TOKENIZERS_PARALLELISM']='false'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9b510fd7",
+   "metadata": {},
+   "source": [
+    "## Importing and pre-processing the IMDB Reviews Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "477314ec",
+   "metadata": {},
+   "source": [
+    "Here we load the IMDB reviews datasets and use a pretrained bert model to tokenize the text."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "08d17b64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train = tfds.load('imdb_reviews', split='train', shuffle_files=True)\n",
+    "val = tfds.load('imdb_reviews', split='test', shuffle_files=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2116803",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# using only the first 1000 datapoints to reduce execution time\n",
+    "\n",
+    "train_df = tfds.as_dataframe(train)[:1000].copy()\n",
+    "train_df['text'] = train_df['text'].apply(lambda x: x.decode('utf-8'))\n",
+    "\n",
+    "val_df = tfds.as_dataframe(val)[:1000].copy()\n",
+    "val_df['text'] = val_df['text'].apply(lambda x: x.decode('utf-8'))"
+   ]
+  },
```

Comment:
If easy, would be nice to display first 3 examples from train data and val data, but no worries if not.